src/ex3/ItemManager.js

```diff
@@ -0,0 +1,99 @@
+import { PokemonClient } from './PokemonClient.js';
+import chalk from 'chalk';
+import * as fs from 'fs';
+
+export async function addTodo(todo) {
+  writeToFile(`${todo}\n`, 'Task was not added', 'Task was added succesfuly!', {
+    flags: 'a',
+  });
+}
+
+
+
+export async function addPokemon(id) {
+  const pokemonClient = new PokemonClient();
+  const result = await pokemonClient.fetchSinglePokemon(id);
+
+  if (!result) {
+    console.log(chalk.red(`could not fetch the pokemon ${id}.`));
+    return;
+  }
+
+  writeToFile(
+    `Catch ${result.name}\n`,
+    'Pokemon was not added',
+    'Pokemon was added succesfuly!',
+    { flags: 'a' }
+  );
+}
+
+
+
+export async function addPokemons(pokemons) {
+  const pokemonClient = new PokemonClient();
+  const result = await pokemonClient.fetchPokemons(pokemons);
+
+  if (!result) {
+    console.log(`could not fetch one of the pokemons ${pokemons}`);
+    return;
+  }
+
+  result.forEach((pokemon) => {
+    writeToFile(
+      `Catch ${pokemon.name}\n`,
+      'Pokemon was not added',
+      'Pokemon was added succesfuly!',
+      { flags: 'a' }
+    );
+  });
+}
```

Comment:
it looks like you have a bug when inserting a name that doesn't represent a pokemon
<img width="1233" alt="image" src="https://user-images.githubusercontent.com/9549288/173354871-c8ce3ddb-922e-42b9-bd18-9383be39418d.png">

Reply:
It happens because fetchPokemons throws an error. I changed it to only print the error message.